js/modules/messages_data_migrator.js

```diff
@@ -65,9 +65,9 @@ exports.processNext = async ({
 
   const totalDuration = Date.now() - startTime;
   const numProcessed = messagesRequiringSchemaUpgrade.length;
-  const hasMore = numProcessed > 0;
+  const done = numProcessed < count;
```

Comment:
I don't know what the `count` variable represents, and therefore this line seems suspect to me. Maybe we need a better variable name?

Reply:
Good call! Agreed, I’ll change it.

Reply:
Will use `numMessagesPerBatch`.